README.md

```diff
@@ -14,6 +14,8 @@ Smithay aims to provide building blocks to create wayland compositors in Rust. W
 being a full-blown compositor, it'll provide objects and interfaces implementing common
 functionalities that pretty much any compositor will need, in a generic fashion.
 
+It tries to support some commonly used Wayland protocol extensions, such as relevant ones made by [wlroots](https://gitlab.freedesktop.org/wlroots/wlroots) for use with docks and standalone window managers. See https://github.com/Smithay/smithay/issues/778 for more information.
+
```

Comment:
Not that I'm saying that this is phrased wrong,
but I think mentioning that we support [wayland core](https://gitlab.freedesktop.org/wayland/wayland) protocols (~5 in total), and [official protocol extensions](https://gitlab.freedesktop.org/wayland/wayland-protocols) (~11 in total) should go first before mentioning that we support some unofficial [wlr-protocols](https://gitlab.freedesktop.org/wlroots/wlr-protocols) (1 I think) and [plasma-wayland-protocols](https://github.com/KDE/plasma-wayland-protocols/) (also 1) extensions.

My point is that this official extensions are way more prevalent in the codebase, so it sounds weird to list only wlroots here.

Reply:
"We support the Wayland core protocols, the official protocol extensions, and some unofficial extensions, such as those made by and for wlroots and KDE"

How does that sound?

Reply:
Nice, that's exactly what I was looking for!

Reply:
Give it another look, and tell me what you think. I also included a (hidden) comment that links back to this so people reading it can come here and see what the situation was like as of the time of writing.